calculator.rb

```diff
@@ -0,0 +1,26 @@
+puts "Welcome to your calculator. Choose from the following operations: \n 1. add (+) \n 2. subtract (-) \n 3. multiply (*) \n 4. divide (/) \n Please type the name or symbol for the operator you would like to use"
+
+chosen_operator = gets.chomp!
+
+#gets numbers from user
+  puts "First number:"
+  first_number = gets.chomp.to_f
+  puts "Second number:"
+  second_number = gets.chomp.to_f
+
+#checking if number is integer by dividing it by itself
+  if first_number / first_number == 1 && second_number / second_number == 1
+
```

Comment:
This is a creative way to address this problem. Unfortunately it's got one big issue: it doesn't work for 0!